libminifi/test/keyvalue-tests/PersistableKeyValueStoreServiceTest.cpp

```diff
@@ -33,17 +33,14 @@
 
 static std::string config_yaml; // NOLINT
 
-static inline void configYamlHandler(Catch::ConfigData&, const std::string& path) {
-  config_yaml = path;
-}
-
 int main(int argc, char* argv[]) {
   Catch::Session session;
 
-  auto& cli = const_cast<Catch::Clara::CommandLine<Catch::ConfigData>&>(session.cli());
-  cli["--config-yaml"]
-      .describe("path to the config.yaml containing the PersistableKeyValueStoreService controller service configuration")
-      .bind(&configYamlHandler, "path");
+  const auto& const_cli = session.cli();
+  auto& cli = const_cast<minifi::utils::remove_cvref_t<decltype(const_cli)>&>(const_cli);
```

Comment:
Is it worth all this code just to avoid a copy (in a test?)  The example at https://github.com/catchorg/Catch2/blob/devel/docs/own-main.md#adding-your-own-command-line-options has
```c++
auto cli = session.cli() | ...;
session.cli(cli);
```

Reply:
Good point. I only read the clara docs, not the one you linked. I'll rewrite this part to avoid this ugly `const_cast`.

Reply:
done in f4c9ebb